src/services/user/user-router.ts

```diff
@@ -40,14 +42,23 @@ userRouter.get(
             },
         },
     }),
-    (req, res) => {
+    async (req, res) => {
         const payload = getAuthenticatedUser(req);
-        const token = generateJwtToken(payload, false, Config.QR_EXPIRY_TIME);
-        const uri = `hackillinois://user?userToken=${token}`;
-        return res.status(StatusCode.SuccessOK).send({ userId: payload.id, qrInfo: uri });
+        const currentTime = Math.floor(Date.now() / Config.MILLISECONDS_PER_SECOND);
+        const exp = currentTime + Config.QR_EXPIRY_TIME_SECONDS;
+
+        // Encrypt user ID and expiration timestamp
+        const encryptedToken = encryptQR(payload.id, exp);
+
+        const uri = `hackillinois://user?qrId=${encryptedToken}`;
```

Comment:
Instead of having all of this logic in user-router and duplicated in staff router, we should have `encryptQR` just take in user id. Then, return a QR (with uri) that expires in QR_EXPIRY_TIME_SECONDS.